stock_request/models/stock_request_order.py

```diff
@@ -111,6 +112,23 @@ def _get_default_requested_by(self):
          'Stock Request name must be unique'),
     ]
 
+    direction = fields.Selection([('outbound', 'Outbound'),
```

Comment:
Have you thought to split this changes in a new module? It would be fine to keep stock_request as simple as it can be and delegate specific features to dependent modules.

Just wondering

Reply:
@hveficent We have spent quite a bit of extra time on this pull request. Let's have this discussion for the v13 migration. Please create an issue.

Reply:
Ok, but then, direction fields seems to be out of the scope of this PR. What is used for?

Reply:
It allows the user to specify if the product needs to be shipped to the external location or brought back to the warehouse. Based on the direction, the inventory location is set.